### DISCUSSION TITLE
Cannot adjust Start/End/ViaPoints by mouse when I planning a route (Desktop version)

### DISCUSSION THREAD

**Discussion by qyb** [devemux86/cruiser#144] [Q&A] [answered]
I still keep version 3.0.2 for this feature. Is there another keyboard shortcut to drag&drop them?

**Comment by devemux86** [comment, marked as answer]
Hi @qyb,

To move waypoints or favorites press:
- **Space** or **Caps Lock** + **mouse drag**

---

All keyboard shortcuts are listed here:
- https://github.com/devemux86/cruiser/discussions/42
- https://wiki.openstreetmap.org/wiki/Cruiser
